test/end-to-end/pages/will/codicils.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 
-const config = require('config');
-const commonContent = require('app/resources/en/translation/common');
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
+const pagePath = require('app/steps/ui/will/codicils');
 
```

Comment:
This is undoing performance work - including these files takes time - do we need to?